### ISSUE TITLE
为什么使用 self.discriminator.zero_grad() 和 self.vqgan.zero_grad()？

### ISSUE THREAD

**Issue by XudongHan-bio** [aa1234241/vqgan#3]
Hello, why do you use self.discriminator.zero_grad() and self.vqgan.zero_grad() for training? Won't this prevent gradient descent?

**Comment by aa1234241** [comment]
Hi, you can refer to [this](https://stackoverflow.com/questions/48001598/why-do-we-need-to-call-zero-grad-in-pytorch)
